src/server/master_svr_attr_def.xml

```diff
@@ -1469,8 +1469,8 @@
 	</member_verify_function>
    </attributes>
    <attributes>	
-   /* SRV_ATR_JobRequeueTimeout */
-	<member_name><both>ATTR_job_requeue_timeout</both></member_name>		<!-- "job_requeue_timeout" -->
+   /* SRV_ATR_JobRequeueDelay */
```

Comment:
Now that we are changing this, all instance of SRV_ATR_JobRequeTimeout has to change too.  So, please change server.h and req_rerun.c to make use of the new enum.